Xamarin.Forms.Platform.GTK/Renderers/WebViewRenderer.cs

```diff
@@ -203,6 +206,15 @@ private void OnEvalRequested(object sender, EvalRequested eventArg)
             }
         }
 
+        Task<string> OnEvaluateJavaScriptRequested(string script)
+        {
+	       if (Control != null)
+	       {
+		       Control.ExecuteScript(script);
+	       }
+               return null;
```

Comment:
Should we always return null?

Reply:
Did this per @jassmith's recommendation in the issue thread: https://github.com/xamarin/Xamarin.Forms/issues/1699